src/sql/workbench/contrib/charts/browser/chartView.ts

```diff
@@ -86,7 +86,7 @@ export class ChartView extends Disposable implements IPanelView {
 	public readonly onOptionsChange: Event<IInsightOptions> = this._onOptionsChange.event;
 
 	constructor(
-		private readonly _renderOptionsInline: boolean,
+		private readonly _isQueryEditorChart: boolean,
```

Comment:
I feel like these should really be two separate actions in case someone else wanted to use this in the future but not have the options shown inline.

That being said that could just be work we do at that point instead so I don't really have a huge issue with this current implementation.